src/handwritten/__init__.py

```diff
@@ -1,7 +1,6 @@
 from nidaqmx.errors import DaqError, DaqReadError, DaqWriteError, DaqWarning, DaqResourceWarning
 from nidaqmx.scale import Scale
 from nidaqmx.task import Task
-from nidaqmx._task_modules.read_functions import CtrFreq, CtrTick, CtrTime
```

Comment:
These are public aliases for clients to use. Don't remove them. Instead, change them to import from the correct submodule, `nidaqmx.types`.